inbm/dispatcher-agent/dispatcher/sota/snapshot.py

```diff
@@ -157,6 +163,26 @@ def take_snapshot(self) -> None:
             self._dispatcher_callbacks.broker_core.telemetry("SOTA snapshot succeeded")
             self.snap_num = snapshot_num
 
+    def delete_snapshot(self) -> int:
+        """Deletes snapshots from the directory"""
```

Comment:
Snapshots are not deleted from a directory.. configs are stored in that directory. I would suggest that the configs be manipulated through the snapper tool and not through the directory, as this could change in the future. You can manipulate or list configs with snapper -- see https://man.archlinux.org/man/snapper.8